<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>exampleClass</class>
 <widget class="QWidget" name="exampleClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>893</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>example</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2,1">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <property name="topMargin">
        <number>120</number>
       </property>
       <item row="0" column="0">
        <widget class="QWebView" name="webView_pie">
         <property name="url">
          <url>
           <string>about:blank</string>
          </url>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWebView" name="webView_bar">
       <property name="url">
        <url>
         <string>about:blank</string>
        </url>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget_ia_warn"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2,1">
     <item>
      <widget class="QWebView" name="webView_line">
       <property name="url">
        <url>
         <string>about:blank</string>
        </url>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_map" native="true"/>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget_warn"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="example.qrc"/>
 </resources>
 <connections/>
</ui>
